core/regression.py

```diff
@@ -65,6 +47,23 @@ def kmeans_run(run_df, regressor, estimator):
             clustered_test_data['result']=model[i].predict(clustered_test_data.drop('trace_id',1))       
     return clustered_test_data['result']
 
+def regression(test_df, job, model):
+    split = model['split']
+    test_data, original_test_data = prep_data(test_df)
+    if split['type'] == 'single':
+        regressor = joblib.load(split['model_path'])
+    elif split['type'] == 'double':
+        regressor = joblib.load(split['model_path'])
+        estimator = joblib.load(split['kmean_path'])
+        
+    if job['clustering'] == KMEANS:
+        results_df = kmeans_clustering(original_test_data, regressor, estimator)
+    else:
+        results_df = no_clustering(original_test_data, test_data, regressor)
+
+    results = prepare_results(results_df)
+    return results
+
 def kmeans_clustering(original_test_data, regressor, estimator):
     print(estimator)
```

Comment:
I don't like simple prints in the completed code. It makes it hard to see what is going on. Fine for local development, but remove them for PRs.